src/cf-expandables/src/cf-expandables.less

```diff
@@ -17,14 +17,9 @@
 // .o-expandable_link
 @expandable_link-text:          @pacific;
 
-// .o-expandable__padded
-@expandable__padded-bg:         @gray-10;
-@expandable__padded-bg-hover:   @gray-20;
-@expandable__padded-divider:    @gray-40;
-
-// .o-expandable-group
-@expandable-group-bg:           @white;
-@expandable-group-divider:      @gray-40;
+// .o-expandable modifiers
+@expandable__background:        @gray-5;
```

Comment:
Shouldn't this be `@expandable__gray5-background`.  This would allow you to have different color backgrounds?

Reply:
The background should not be any other color according to the spec.